timesketch/api/v1/resources/contextlinks.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2022 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Context link API for version 1 of the Timesketch API."""
+import logging
+import re
+import copy
+
+from flask import jsonify
+from flask_restful import Resource
+from flask_login import login_required
+
+from timesketch.api.v1 import resources
+from timesketch.api.v1.utils import load_yaml_config
+
+# Set up logging
+logger = logging.getLogger("timesketch.contextlinks_api")
+
+
+class ContextLinkConfigResource(resources.ResourceMixin, Resource):
+    """Resource to get context link information."""
+
+    @login_required
+    def get(self):
+        """Handles GET request to the resource.
+        Returns:
+            JSON object including version info
+        """
+        context_link_config = load_yaml_config("CONTEXT_LINKS_CONFIG_PATH")
+
+        response = {}
+
+        if not context_link_config:
+            logger.warning("The config file 'data/context_links.yaml' is empty!")
```

Comment:
Good point. However, after our talk about what to report in the logs and what not, I will opt for removing the warning. Since the default is an empty config file, this would create unnecessary log entries for a feature that the operator maybe just does not want to use. 
TL;DR: An empty config file is working as intended and hence does not need a warning.